asmcli/asmcli.sh

```diff
@@ -2657,19 +2668,21 @@ EOF
 
 print_config() {
   local MANAGED; MANAGED="$(context_get-option "MANAGED")"
-  local CUSTOM_OVERLAY; CUSTOM_OVERLAY="$(context_get-option "CUSTOM_OVERLAY")"
 
   if [[ "${MANAGED}" -eq 1 ]]; then
     cat "${MANAGED_MANIFEST}"
     return
   fi
 
   PARAMS="-f ${OPERATOR_MANIFEST}"
-  while read -d ',' -r yaml_file; do
-    PARAMS="${PARAMS} -f ${yaml_file} "
+  while read -r yaml_file; do
+    if [[ -n ${yaml_file} ]]; then
+      PARAMS="${PARAMS} -f ${yaml_file} "
+    fi
   done <<EOF
-${CUSTOM_OVERLAY}
+$(context_list-istio-yamls)
 EOF
+  echo "${PARAMS}"
```

Comment:
Is this a debug statement or here on purpose?

Reply:
Sorry. Debug params.